modules/behavioural_qc/php/views/conflicts.class.inc

```diff
@@ -27,15 +27,23 @@ class Conflicts
      * @param array $visits      The visits for form element
      * @param array $instruments The instruments for form element
      * @param array $projects    The projects for form element
+     * @param array $subprojects The subprojects for form element
      * @param array $sites       The sites for form element
      */
-    public function __construct($visits, $instruments, $projects, $sites)
-    {
+    public function __construct(
+        $visits,
+        $instruments,
+        $projects,
+        $subprojects,
+        $sites
+    ) {
         $this->_formElement = [
             'instruments' => $instruments,
             'visits'      => $visits,
             'projects'    => $projects,
+            'subprojects' => $subprojects,
             'sites'       => $sites,
+            'subprojects' => $subprojects,
```

Comment:
You seem to have added subprojects twice here.

Reply:
Thanks!